docs/start/getting-started/fragments/angular/data-model.md

```diff
@@ -61,7 +60,7 @@ amplify push
 
 ### Code generation
 
-Since you added an API the `amplify push` process will automatically prompt for codegen (select `y` when prompted for automatic query and code generation). Choose **Angular** which will create an `API.service.ts` file in the app directory.
+Since you added an API the `amplify push` process will automatically prompt for codegen (select `yes` when prompted for automatic query and code generation). Choose **Angular** which will create an `API.service.ts` file in the app directory.
```

Comment:
```suggestion
Because you added an API, the `amplify push` process will automatically prompt for codegen. Select `yes` when prompted for automatic query and code generation. Choose **Angular**, which will create an `API.service.ts` file in the app directory.
```